logrotate_unix.go

```diff
@@ -1,3 +1,5 @@
+//go:build !windows
```

Comment:
by default it runs only for unix

Reply:
I am not sure I understand you.

Reply:
`go:build` tag is useless here

Reply:
To my knowledge, `unix` is not a special filename suffix for Go.

Anyway. I reverted previous FreeBSD code because currently don't have an easy way to test that my changes work on FreeBSD.

Reply:
the suffix `_unix.go` [was supported then dropped](https://go-review.googlesource.com/c/go/+/426296) in favor of unix build tag. 

IMO `//go:build unix` is safer

Reply:
https://github.com/fahedouch/go-logrotate/blob/main/go.mod#L3

Reply:
https://github.com/fahedouch/go-logrotate/blob/main/.github/workflows/test.yml#L33

How this works at all...

Reply:
ah not good! lets make a PR to fix this before ^^